src/main/java/mekanism/common/integration/IMCHandler.java

```diff
@@ -24,105 +21,55 @@ public void onIMCEvent(List<IMCMessage> messages)
 		{
 			if(msg.isNBTMessage())
 			{
-				try {
-					boolean found = false;
-					boolean delete = false;
-					
-					String message = msg.key;
-					
-					if(message.equals("ShapedMekanismRecipe"))
```

Comment:
I was thinking about this tonight, perhaps we should log an error for the unlikely case that someone's using this? Rather than just silently failing